lib/plugins/command_block.js

```diff
@@ -8,9 +8,11 @@ function inject (bot) {
     assert.strictEqual(bot.player.gamemode, 1, new Error('The bot has to be in creative mode to open the command block window'))
     assert.notStrictEqual(pos, null)
     assert.notStrictEqual(command, null)
-    assert.strictEqual(bot.blockAt(pos).name.includes('command_block'), true, new Error("The block isn't a command block"))
-
+    if (options.checkblock !== false) {
```

Comment:
that is the wrong fix

the check is correct, the fix is to call that function at a later time when the bot got the information (using the block update event for example)